terraform/modules/data-share-service/waf.tf

```diff
@@ -208,6 +208,34 @@ resource "aws_wafv2_web_acl" "load_balancer" {
     allow {}
   }
 
+  rule {
+    action {
+      block {}
+    }
+
+    name     = "SqlInjectionMatch"
+    priority = 1
+
+    statement {
+      sqli_match_statement {
+        text_transformation {
+          type     = "URL_DECODE"
+          priority = 1
+        }
+
+        field_to_match {
+          query_string {}
```

Comment:
Does query_string match request bodies too? I feel like on our POSTs, this is worth doing

Reply:
Done